src/meltano/core/plugin/file.py

```diff
@@ -1,3 +1,5 @@
+import structlog
```

Comment:
@WillDaSilva I may have missed in other PRs, but I notice `from __future__ import annotations` is not added to all files. Does that mean it's being added manually?

If so, I think we can fix that by adding the import as part of an `isort` fix

```toml
[tool.isort]
add_imports = [
    "from __future__ import annotations",
]
```

Wdyt?

Reply:
That's a cool `isort` feature. It'll prevent us from accidentally re-introducing the old annotation style, and eventually (many years from now) once `from __future__ import annotations` is not needed for any Python version we support, `pyupgrade` will remove it from all of our files automatically.

I've added that line to the `isort` config in:
- https://github.com/meltano/meltano/pull/6546